pkg/cluster/backup_manager.go

```diff
@@ -165,8 +165,23 @@ func (bm *backupManager) checkBackupExist(ver string) (bool, error) {
 	return bm.bc.Exist(ctx, ver)
 }
 
-func (bm *backupManager) getStatus() (*backupapi.ServiceStatus, error) {
+func (bm *backupManager) getStatus() (*spec.BackupServiceStatus, error) {
```

Comment:
the backup manager should still return backuapi related thing. we should convert it when we put it into tpr